packages/eds-core-react/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -250,7 +250,7 @@ export type AutocompleteChanges<T> = { selectedItems: T[] }
 
 export type AutocompleteProps<T> = {
   /** List of options in dropdown */
-  options: T[]
+  options: readonly T[]
```

Comment:
looks like this needs some more work for typescript to be happy
![bilde](https://github.com/equinor/design-system/assets/4499531/c9acca75-ca9d-41b6-ad23-baa3d9fe7608)

Reply:
I will fix this (adding readonly) in another pr for #3514. By the way looks like `downshift `is happy again if I add `as T[]` behind `availableItems`